src/core/evaluator.js

```diff
@@ -708,24 +708,39 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
           new ErrorFont('Font ' + fontName + ' is not available'), font));
       }
 
-      var fontRef, xref = this.xref;
+      let fontRef, fontRes, xref = this.xref;
       if (font) { // Loading by ref.
         if (!isRef(font)) {
           throw new Error('The "font" object should be a reference.');
         }
         fontRef = font;
       } else { // Loading by name.
-        var fontRes = resources.get('Font');
+        fontRes = resources.get('Font');
         if (fontRes) {
           fontRef = fontRes.getRaw(fontName);
         } else {
-          warn('fontRes not available');
-          return errorFont();
+          warn(`fontRes "${fontName}" not available.`);
+          // Attempting to fallback, see the code-path just below...
```

Comment:
I think we should put this in the warning here/below instead (or both) so it's clear from the console that this is a recovery path. We did this before for some other recovery messages.

Reply:
Good point; I've removed the (frankly) unnecessary warning duplication, and also added different warning messages depending on whether the fallback/Error paths are taken.